pkg/scheduler/scheduler.go

```diff
@@ -269,53 +265,35 @@ func (s *Scheduler) requeueClusterResourceBindings(selector labels.Selector) err
 
 func (s *Scheduler) getPlacement(resourceBinding *workv1alpha1.ResourceBinding) (policyv1alpha1.Placement, string, error) {
 	var placement policyv1alpha1.Placement
-	var clusterPolicyName string
-	var policyName string
-	var policyNamespace string
-	var err error
-	if clusterPolicyName = util.GetLabelValue(resourceBinding.Labels, policyv1alpha1.ClusterPropagationPolicyLabel); clusterPolicyName != "" {
-		var clusterPolicy *policyv1alpha1.ClusterPropagationPolicy
```

Comment:
this may be a risk as I removed the `clusterPolicyName`, but since the variable is `ResourceBinding` not `ClusterResourceBinding`, i think it will never has change to run. Correct me if not. Thanks.